rust/perspective-viewer/src/rust/components/column_settings_sidebar/style_tab/column_style.rs

```diff
@@ -163,64 +162,8 @@ pub fn ColumnStyle(p: &ColumnStyleProps) -> Html {
         _ => Err("Booleans aren't styled yet.".into()),
     };
 
-    let props = p;
-    let plugin = props
-        .renderer
-        .get_active_plugin()
-        .expect("No active plugins!");
-
-    let plugin_column_names = plugin
-        .config_column_names()
-        .map(|arr| {
-            arr.into_serde_ext::<Vec<Option<String>>>()
-                .expect("Could not deserialize config_column_names into Vec<Option<String>>")
-        })
-        .unwrap_or_default();
-
-    let view_config = props.session.get_view_config();
-    let mut is_symbol = false;
-    let components = plugin_column_names
-        .iter()
-        .zip(&view_config.columns)
-        .filter_map(|(ty, name)| {
-            let attr_type = props
-                .session
-                .metadata()
-                .get_column_view_type(&props.column_name)
-                .expect("Couldn't get column type for {column_name}");
-
-            let zipped = ty.as_deref().zip(name.as_deref());
-            match zipped {
-                Some(("Symbol", name)) if name == props.column_name && p.ty == Type::String => {
-                    is_symbol = true;
-                    Some(html! {
-                        <SymbolAttr
-                            { attr_type }
-                            column_name={ props.column_name.clone() }
-                            session={ &props.session }
-                            renderer={ &props.renderer }
-                            custom_events={ &props.custom_events }/>
-                    })
-                },
-                _ => None,
-            }
-        })
-        .collect::<Html>();
-
-    let is_ok = opt_html.is_ok();
     html_template! {
         <LocalStyle href={ css!("column-style") } />
-        if is_ok {
-            { opt_html.unwrap() }
-        }
-
-        { components }
-        if !is_ok && !is_symbol {
-            <div class="style_contents">
-                <div id="column-style-container" class="no-style">
-                    <div class="style-contents">{ "No styles available" }</div>
-                </div>
-            </div>
-        }
+        {opt_html.expect_throw("Could not generate HTML to style this column!")}
```

Comment:
Errors are not the right way to handle this (or really in general), not even JavaScript errors. It is _really_ hard to predict when something that is the result of a long complex process will adhere to restriction like this that cannot (or at least, isn't) be represented in the type system. More importantly, throwing an error here can corrupt/crash the rust app.

There is a similar circumstance in [`active_column.rs`](https://github.com/finos/perspective/blob/master/rust/perspective-viewer/src/rust/components/column_selector/active_column.rs#L394), where there is a small corner case where the `View` must exists before the UI can be rendered. This render path shouldn't be hit, so we render a stub.

In this case, we probably want to keep the stub, we just don't want it ever to show. If for some reason it does, we don't want to segfault, so it needs to render something because we can't (AFAIK) prevent this logic from being called with a missing view type (?). Probably also worth warning the console as this is programmer error.